org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/IRunnableDevice.java

```diff
@@ -119,19 +119,15 @@ default void run() throws ScanningException, InterruptedException, TimeoutExcept
 	 * @throws InterruptedException
 	 */
 	default void start(final IPosition pos) throws ScanningException, InterruptedException, TimeoutException, ExecutionException {
-
 		final List<Throwable> exceptions = Collections.synchronizedList(new ArrayList<>(1));
-		final Thread thread = new Thread(new Runnable() {
-			@Override
-			public void run() {
-				try {
-					IRunnableDevice.this.run(pos);
-				} catch (ScanningException | InterruptedException | TimeoutException | ExecutionException e) {
-					// If you add an exception type to this catch clause,
-					// you must also add an "else if" clause for it inside
-					// the "if (!exceptions.isEmpty())" conditional below.
-					exceptions.add(e);
-				}
+		final Thread thread = new Thread(() -> {
+			try {
+				IRunnableDevice.this.run(pos);
+			} catch (ScanningException | InterruptedException | TimeoutException | ExecutionException e) {
+				// If you add an exception type to this catch clause,
+				// you must also add an "else if" clause for it inside
+				// the "if (!exceptions.isEmpty())" conditional below.
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)